test/test_casatable.py

```diff
@@ -7,12 +7,12 @@
 from sourcefinder.testutil.decorators import requires_data
 from .conftest import DATAPATH
 
-casatable = os.path.join(DATAPATH, 'aartfaac.table')
+casatable = os.path.join(DATAPATH, 'accessors/casa.table')
```

Comment:
Can we name the dataset something meaningful? `casa.table` is too generic, and gives no idea about what is different about this dataset as opposed to the other ones.

Reply:
Right. But it will be hard for me to find out what kind of observational data is in `casa.table`, i.e. it will be hard to make the description more specific.

But I guess this test is about access to a certain format of data, i.e. its contents are not important.
The casa data format itself should be described in a document on the NRAO website.